src/views/CreateShareStructure.vue

```diff
@@ -26,7 +26,7 @@
         of shares, each class must be assigned an identifying name such as Class A, Class B, etc.
       </p>
       <div id="share-structure-sample">
-        <ListShareClass :shareClasses="sharesHelpSample"/>
+        <ListShareClass :shareClasses="sharesHelpSample" :isSummary="true"/>
```

Comment:
I noticed this component, even with isSummary=true, still highlights rows when you mouse over them. You may need to disable cursor events in this case.

Reply:
That is the data table feature and I am not sure whether it is a bug or it should be disabled when  it is is summary mode. If it needs to be done, I will have to know a bit more about data table and the deep selectors that can be used in data table I think. Probably in another ticket?

Reply:
It's already described in /bcgov/entity#3882 :)

Reply:
Yes I saw that. All other points in the ticket are resolved as part of this PR and this PR has a P1 fix as well. So doing more rearch on data table should not block this PR as such

Reply:
Fair enough. How about updating that ticket's text and putting it back into the backlog?

Reply:
I prefer to create another ticket as that one has to look into how to disable fetaures like hover styles, pointer style etc. Before that I would like to confirm with PO whether this is actually an issue

Reply:
OK.

Reply:
Checked with Linda. She confirmed that the data fix and the edit issues need to be resolved and the highlighting is not an issue